### DISCUSSION TITLE
洗澡的時候總是腦洞大開？

### DISCUSSION THREAD

**Discussion by tonyYZL** [1101-GS4719-C/Python-Community-GS4719-C#1443] [General]
我最近被一個程式作業困擾了好幾天，怎麼寫都是錯的，不然就是效能很差
結果某一天洗澡的時候突然冒出一個想法，我就趕快去試
結果就成功了！而且效能還超高

這時我突然想到，很久以前有聽說過洗澡的時候靈感都會很多，我這次經驗也完美的驗證這個說法，於是我出於好奇上網找了相關文章，找到這段敘述：

「然而當我們去洗澡的時後，這些思緒會被打斷，讓大腦可以隨意漫遊。洗澡時大腦會放鬆前額葉皮質，並且進入預設模式網絡（DMN），允許跳脫脈絡的隨機思考方式，而這也是我們往往在澡堂裡靈感爆發的原因。」
(擷取自 https://www.cw.com.tw/article/5099638)

我覺得很有趣，洗澡跟思考看似毫無關聯的兩件事，卻對思考有這麼特別的影響，問了幾個朋友之中有過半數也都有同樣的感受

你/妳洗澡的時候也會靈感爆棚嗎？
知道以後遇到問題該怎麼辦了嗎？

不說了，我要去洗澡了

**Comment by TW-Maverick** [comment]
對你分享文章中的這段話深有同感，『這不是洗澡的專利，釣魚或是運動也都可以產生同樣的效果。在這些不需經過太多思緒的活動中，你的大腦會進入自動導航。』當寫程式或專題遇到困難時，我會試著改變處在的環境，比如說躺在床上呈現放空的狀態下或者散散步，我覺得這些都是可以讓人富有新想法。

**Comment by ThanatosJun** [comment]
靈光一閃!!真的很多創世巨作就是這麼出現了~

**Comment by sjh501** [comment]
我也常常很煩躁或找不到出路的時候去洗個澡，尤其是冷水澡，常常會想通很多事情，或是一些普物的題目會突然有個解法。大推～

**Comment by pigeon9975** [comment]
我寫題目寫到卡住的時候也會選擇先去洗澡，不僅可以轉換心情，也可以讓自己暫時脫離很躁的環境，不要浪費太多時間在糾結同一件事上，不過在浴室根本靈感爆棚，從小到大有好多好多難題都是在浴室想出來的，洗澡時想東西真的大推！！！

**Comment by alicia0725** [comment]
我也會在很煩躁的時候去洗澡，尤其是洗頭，水從頭上淋下來那一瞬間有種可以打斷我的煩惱，沖毀思考的感覺，而且每次洗完澡都會特別清醒做什麼事情比較不會迷糊！

**Comment by Alexgns** [comment]
洗澡轉換心情非常有效，而且會讓人變得更有精神，我在作業做不出來時也都會選擇洗澡休息一下，不僅變得更清醒，思路也更清晰

**Comment by alstinlin** [comment]
我也有過類似的經驗! 當時也是程式一籌莫展，但是洗澡的時候突然有了一個新的想法，後來的實作過程也很順利，沒想到換了想法問題會迎刃而解，考慮之後打程式前先洗個澡xd

**Comment by pudding0803** [comment]
我很常會在洗澡的時候思考很多事情，所以一不小心就會洗超久🥹，不過通常都可以想通很多問題或思考出人生的新方向(?)，如果心情不好的話，洗完澡之後也會變得比較快樂🥳，推薦沒試過的人可以試試看，放慢洗澡的步調😇！

**Comment by zephyrxin01** [comment]
阿基米德就是在洗澡的時候發現浮力原理的，可見洗澡對大家來說不只能放鬆，還能激發想像力。而且洗完澡後全身舒爽也能讓自己心情愉悅。

**Comment by RogelioKG** [comment]
一直很難理解室友的五分鐘戰鬥澡。
洗澡是唯一很放鬆而且又可以自己獨處的場合，
而且總感覺洗澡時候的思維和一般情況下不同，
這時候拿來想一些程式問題真的是特別的合適。
有時候就會不小心洗的有點久。

**Comment by lin157644** [comment]
洗澡真的很神奇，當下思緒會莫名的清晰，洗完後精神也會比較好，我感覺這個世代一天都在不斷的接收資訊，只有洗澡的時間我們是遠離資訊設備、遠離人、事的狀態，因此有辦法全心全意的處理這些資訊。
